src/uproot/extras.py

```diff
@@ -270,3 +270,20 @@ def dask():
         )
     else:
         return dask, da
+
+
+def dask_awkward():
+    """
+    Imports and returns ``dask_awkward``.
+    """
+    try:
+        import dask_awkward
+    except ModuleNotFoundError:
+        raise ModuleNotFoundError(
+            """for uproot.dask, install the (complete) 'dask' package with:
+    pip install "dask[complete]"
+or
+    conda install dask"""
```

Comment:
This error text doesn't make sense here. Installing `dask[complete]` won't install `dask_awkward`.